### PR TITLE
docs: add code of conduct, contributing and update readme

### PR DESCRIPTION
**Pull request by xixas**
## Summary by CodeRabbit

- **Documentation**
  - Added a Contributor Covenant Code of Conduct to outline community behavior standards.
  - Introduced guidelines for contributing to Osmosys Form Builder, covering questions, bug reporting, feature requests, submission guidelines, development setup, coding rules, and commit message standards.
  - Updated README to provide an overview of OsmosysForm, highlighting its key features, such as dynamic form generation, validations, automated submission handling, custom styling, conditional logic, and reusable components for Angular applications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Installation and Usage
+
+[Publishing Library](projects/osmosys-form/README.md)
+
+TODO: Add detailed installation and usage instructions here.
+
+For more help on the Angular CLI, use `ng help` or check out the [Angular CLI Overview and Command Reference](https://angular.dev/tools/cli) page.
+
+##
```
[the remark this answers is not in the document]
Comment (xixas) [reply README.md:67]: No no, this should be removed create a commit to remove the line instead

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+individual, or aggression toward or disparagement of classes of individuals.
+
+**Consequence**: A permanent ban from any sort of public interaction within
+the community.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
```
[the remark this answers is not in the document]
Comment (xixas) [reply CODE_OF_CONDUCT.md:119]: @coderabbitai create commit suggestion

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (xixas) [reply README.md:67]: @coderabbitai  Where is the commit suggestion?

File: README.md [hunk 3]
```diff
@@ -1,27 +1,66 @@
...
+
+We appreciate the time and effort put in by all contributors to make this project better!
+
+---
+
+## Installation and Usage
+
+[Publishing Library](projects/osmosys-form/README.md)
+
+TODO: Add detailed installation and usage instructions here.
```
[the remark this answers is not in the document]
Comment (osm-vishnukyatannawar) [reply README.md:63]: TODO is added we will add it later.

**Pull request merged**